code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -35,6 +35,8 @@
 	var/datum/weakref/core
 	///The minimum health we can have
 	var/minimum_health = -300
+	///Traits given when going incorporeal
+	var/list/incorporeal_traits = list(TRAIT_PASS_LOW_STRUCTURE, TRAIT_PASS_MOB, TRAIT_PASS_XENO)
```

Comment:
can you double check some of the other cases as well to see if they should be lists like this instead of using the helper

Reply:
The only time the helper is used is when it needs to get traits from a variable list of pass_flags (including hovering above)